src/btree/bt_cursor.c

```diff
@@ -1096,9 +1096,9 @@ __wt_btcur_remove(WT_CURSOR_BTREE *cbt, bool positioned)
 
     if (btree->type == BTREE_ROW) {
         ret = __cursor_row_search(cbt, false, NULL, NULL);
+        WT_ERR_NOTFOUND_KEEP(ret);
```

Comment:
These should probably become the same form that `WT_RET` and `WT_ERR` take. So here, remove line 1098 and the new line becomes:
```
WT_ERR_NOTFOUND_KEEP(__cursor_row_search(...));
if (ret == WT_NOTFOUND)
...
```

Reply:
I've applied this change to all of the places that have this calling sequence (including the one @agorrod flagged, below).